services/permits.service.ts

```diff
@@ -171,14 +178,15 @@ const PERMIT_ACTION_PAGINATION_PARAMS = {
         },
       },
       permitSpecies: {
+        virtual: true,
         type: 'array',
-        raw: true,
-        items: {
-          type: 'object',
-          properties: {
-            id: 'number',
-            name: 'string',
-            nameLatin: 'string',
+        populate: {
```

Comment:
Kai kurie leidimai turi gyvūnų rūšis kokiu tikriausiai duomenų bazėje net nebus. Pvz.: tiesiog beždžionė. Tokiu atveju nebus galima išsaugoti leidimo.  Nebenti tokių leidimų jau nebėra.

Reply:
man Inga sakė, kad naikinti :D  buvau jos klausęs.